scripts/build.ts

```diff
@@ -49,4 +51,4 @@ FS.writeFileSync(
         null,
         2
     )
-);
+);
```

Comment:
Changed for what? I think it would be smart to change the script to bash (at least for now,) because that's basically what it is anyways. The problem is the way it's currently built, it doesn't appear to be easy to simplify without changing its behaviour. It builds `es`, then uses that to generate `cjs`, then copies that to npm, and uses that to generate a webpack bundle. If we don't need either `es` or `cjs` we could build only one plus the final bundle, but this seems like a pretty fine solution the way it's done now.

Reply:
Rollup can do this type of building just fine -- I've had to set it up for declui. _Fixing_ the build system won't work because its very design is bad. I'll look into replacing it.